src/commands/prompt/index.ts

```diff
@@ -231,3 +207,53 @@ function filterOutApiKey(key: string, value: unknown) {
 
   return value;
 }
+
+function handleInputFile(context: SessionContext, inputFile: string) {
+  const filePath = path.resolve(inputFile.replace('~', os.homedir()));
+
+  if (!fs.existsSync(filePath)) {
+    throw new Error(`Couldn't find provided file: ${inputFile}`);
+  }
+
+  const fileContent = fs.readFileSync(filePath).toString();
```

Comment:
Can error be thrown on reading a file? Should we handle error somehow?

Reply:
There is  a top-level `try`/`catch` in `run` prompt command handler